pimpmycause/pimpmycause/templates/profiles/profile.html

```diff
@@ -6,39 +6,93 @@
         <div class="page-register pnl">
             <div class="content-wrap">
                <h1 class="pnl__title">Profile</h1> 
-                <div class="row"> 
-                    <div class="large-12 medium-12 small-12 columns">
-                        <form method="POST" action="">
+                <div class="large-12">
+                    <form method="POST" action="">
+                        <div class="row"> 
+
                             {% csrf_token %}
                             {{ profile_update_form.non_field_errors }}
                             {{ profile_update_form.errors }}
 
-                            <div class="medium-6 large-6 small-12 large-centered small-centered columns">
-                                <div class="row">
-                                    <h2> Hi {{ user.email }}! Please fill out your profile</h2>
+                            {{ user_update_form.non_field_errors }}
+                            {{ user_update_form.errors }}
 
-                                    <div class="small-12 columns">
-                                        {{ profile_update_form.bio.errors }}
-                                        <textarea name="bio" rows=20 required placeholder="Write a short bio">
-                                        {{ profile_update_form.bio.value|default_if_none:"" }}
-                                        </textarea>
-                                    </div>
-                                    <div class="small-12 columns">
-                                        {{ profile_update_form.linkedin.errors }}
-                                        <input type="url" name="linkedin" placeholder="LinkedIn" value='{{ profile_update_form.linkedin.value|default_if_none:"" }}' required>
+                            <h2> Hi {{ user.name }}! Please fill out your profile</h2>
+
+                           <div class="row">
+                                <div class="large-6 medium-6 small-12 columns">
+                                    <div class="large-6 medium-6 small-6 columns">
+                                        {{ profile_update_form.image.errors }}
+
+                                        {% if not profile_update_form.image.value %}
+                                            <img src="https://freeiconshop.com/files/edd/person-flat.png" alt="Profile Image">
+                                            <div class="small-12 columns">
+
+<!--                                           <label for="image"> Upload profile picture </label>
+ -->                                           <input class="button" type="submit" value="Upload Picture">
```

Comment:
For this one can you walk me through what you did so I can try to replicate? 
Did you choose a new file and hit update? note for now it won't show you a previous of new image that has not been implemented